tests/phpunit/TestFramework/Coverage/JUnit/JUnitTestCaseSorterTest.php

```diff
@@ -37,6 +37,8 @@
 
 use Infection\AbstractTestFramework\Coverage\TestLocation;
 use Infection\TestFramework\Coverage\JUnit\JUnitTestCaseSorter;
+use function iterator_to_array;
+use function log;
```

Comment:
unnecessary here no?

Reply:
They're used to confirm the math used to select bucket sort, which is done in this class.